src/components/PeopleAndRoles/OrgPerson.vue

```diff
@@ -226,15 +224,15 @@ export default class OrgPerson extends Mixins(CommonMixin) {
   private orgPerson: OrgPersonIF = null
 
   /** Model value for org/person form validity. */
-  private orgPersonFormValid: boolean = true
+  private orgPersonFormValid = true
 
   // Address related properties
   private inProgressMailingAddress: AddressIF = undefined
   private inProgressDeliveryAddress: AddressIF = undefined
-  private inheritMailingAddress: boolean = true
-  private mailingAddressValid: boolean = false
-  private deliveryAddressValid: boolean = false
-  private reassignCompletingParty: boolean = false
+  private inheritMailingAddress = true
```

Comment:
Wont this type removal give error if strict mode is enabled?

Reply:
It should not give an error because the type is not unknown (re: noImplicityAny). The type is inferred from the assignment.

Reply:
FWIW, in a different project (using Angular and TS Lint), the original code would give an error saying the type was redundant (as it could be inferred from the assignment). So I'm carrying forward that code style.

Reply:
Ok. I wasnt sure

Reply:
Since you brought this up, I tested it by removing the `noImplicitAny` override in tsconfig.json... This led to a bunch of TS errors in our code, so I fixed them! (I will submit those as a separate PR.) There remain some TS errors -- in common and shared components libraries, not the Edit UI code.